src/components/profilePage/AboutStudent/AboutStudentStyles.css

```diff
@@ -0,0 +1,3 @@
+.text{
```

Comment:
dont use this type of names be a little descriptive. This will cause conflicts letter. Correct it everywhere.